transaction-history-manager/src/main/java/com/qpay/transactionhistorymanager/service/TransactionHistoryService.java

```diff
@@ -2,14 +2,18 @@
 
 import com.qpay.libs.models.UserType;
 import com.qpay.transactionhistorymanager.model.dto.TransactionModification;
+import com.qpay.transactionhistorymanager.model.dto.TransactionOutcome;
 import com.qpay.transactionhistorymanager.model.entity.TransactionEntity;
 import org.springframework.data.domain.Pageable;
 
+import java.time.LocalDateTime;
 import java.util.List;
 
 public interface TransactionHistoryService {
 
     List<TransactionEntity> getPageOfTransactionsByUserId(long userId, final Pageable pageable, final UserType userType);
 
     TransactionEntity saveTransaction(TransactionModification transactionModification);
-}
+
+    TransactionOutcome getTransactionsOutcome(long walletId, LocalDateTime startDateTime, LocalDateTime endDateTime);
```

Comment:
When you work on app which is world wide used, you cannot use localDateTime, because every user lives in a different time zone. In that case you should use ZoneDateTime or Instant